doc/getting_started/index.rst

```diff
@@ -153,6 +153,11 @@ Set Up a Toolchain
    <zephyr_sdk>`, which includes toolchains for all supported Zephyr
    architectures.
 
+   In some specific configurations like non-MCU x86 targets on Linux,
+   you may be able to re-use the native development tools provided by
+   your operating system by setting ``ZEPHYR_TOOLCHAIN_VARIANT=host``
+   and not need any toolchain or SDK at all.
```

Comment:
I did see your comment that people don't refer to their host tools as a toolchain, but it seems to me that strictly speaking those are a toolchain:

- https://elinux.org/Toolchains
- https://en.wikipedia.org/wiki/Toolchain

So I would remove ` and not need any toolchain or SDK at all` from this sentence.

Then maybe say "This is likely only to work on Linux host operating systems" to address the comment from @andrewboie. After that I'm +1, and thanks again!

Reply:
OK I will drop the last line, less is more; especially so in a getting started page.

> Then maybe say "This is likely only to work on Linux host operating systems"

Speaking of less is more, I'm reluctant to mention Linux a second time and duplicate the first line in a slightly-less-vague-but-still-vague way.

BTW I understand no one is suggesting an exact and exhaustive enumeration of which configurations work versus not. If someone committed to maintain such a list going forward then it would IMHO not even belong to the getting started page.

@andrewboie is the latest version good enough for you?